packages/xchain-thorchain-amm/src/utils/thornode.ts

```diff
@@ -261,29 +254,41 @@ export class Thornode {
     scheduledQueueItem?: TxOutItem,
     lastBlockHeight?: LastBlock,
   ): Promise<TxStatus> {
-    txStatus.stage = TxStage.OUTBOUND_QUEUED
-    if (scheduledQueue.length == 0 && txData.tx.memo) {
-      // it is not queued, outbound Tx sent
+    txStatus.stage = TxStage.OUTBOUND_CHAIN_UNCONFIRMED
+    console.log(scheduledQueue.length) 
+    if (scheduledQueue.length == 0 && txData?.tx?.memo) {
+      console.log(scheduledQueue.length)
+      // if it is not queued, outbound Tx sent
       txStatus.stage = TxStage.OUTBOUND_CHAIN_UNCONFIRMED
       const pool = txData.tx.memo.split(`:`, 2)
-      const ticker = pool[1].split(`.`, 1).toString()
-      const chain = this.getChain(ticker)
-      txStatus.seconds = this.chainAttributes[chain].avgBlockTimeInSecs
+      const synthCheck = pool[1].split('/')
+      if (synthCheck.length == 2) {
+        console.log(synthCheck.length)
+        console.log(pool)
+        txStatus.seconds = this.chainAttributes[THORChain].avgBlockTimeInSecs
+      } else {
+        console.log(synthCheck.length)
+        console.log(pool)
+        const ticker = pool[1].split(`.`, 1).toString()
+        const chain = this.getChain(ticker)
+        txStatus.seconds = this.chainAttributes[chain].avgBlockTimeInSecs
+      }
```

Comment:
this would be cleaner/safer  if we use assetFromString(),

also remove console.log()